packages/puppeteer-parse/index.js

```diff
@@ -790,7 +790,7 @@ async function retrieveHtml(page) {
           }
         }
       });
-      return document.documentElement.innerHTML;
+      return document.documentElement.outerHTML;
```

Comment:
@jacksonh The change fixed the issue.
With `outerHTML`, we get a <html> element which can be parsed correctly.